### DISCUSSION TITLE
how to allow exactly one website in safe exam browser?

### DISCUSSION THREAD

**Discussion by AndreGismo** [SafeExamBrowser/seb-win-refactoring#679] [Q&A] [answered]
#### Question:
I need to allow exactly one website (a sciebo link, where students can access allowed material for exams). Is this possible?
For example in the [user manual](https://safeexambrowser.org/windows/win_usermanual_en.html#AdditionalResources) I found that it is possible to allow certain websites (as far as I understand this).
So I have added one expression (exactly the allowed URL) in the `Network` tab of the SEB Configuration Tool.
However, I cannot access any website. In fact, I dont even know, where I should enter the URL?
Once I start the Safe Exam Browser, I only have this window that I specified via `Start URL` in the `General` tab of the SEB Configuration Tool. The website specified via `Start URL` is the online portal where students sign in for their exam. The only thing one can enter there is the matriculation number.

When I add edge to the `Applications` tab, then I can start the browser -- fine. However, than I can access any urls. So I guess, this cannot be the solution(?)

---
#### System: 
Edition: Windows 10 PRO
Version: 22H2

#### SEB:
Version 3.4.1

**Comment by dbuechel** [comment]
Once you activate the URL filter, it filters all network traffic within the internal browser of SEB. Thus, you'll need to make sure that all websites your students should visit are actually allowed. You can test the URL filter rules in real time by enabling the live application log under tab "Security", which will then show you exactly which URL is getting blocked and why (as long as you've enabled the logging of URLs, see tab "Browser" > "Show URLs").

**Reply by AndreGismo** [reply]
Thank you for the hint!

Ok, but how can I access the internal browser of SEB?
(so I don't need to separately add edge or some other browser, right?)

**Reply by AndreGismo** [reply]
In the meanwhile I have managed to make an address bar appear atop the browser window where one can enter the URL. When I enter the specified URL, the new page opens (but in the same browser window), so then the old page is gone.

Is it possible to have two separate browser windows? And how can one open an additional browser window in SEB?

**Reply by dbuechel** [reply, marked as answer]
Yes, see: https://safeexambrowser.org/windows/win_usermanual_en.html#BrowserPane > "Links requesting to be opened in a new browser window..."

**Reply by AndreGismo** [reply]
aah I see, thank you!

**Reply by AndreGismo** [reply]
Thank you for all your support @danschlet and @dbuechel ! I have also managed to include the link in the LMS and now it works fine.

**Comment by danschlet** [comment]
> I need to allow exactly one website (a sciebo link, where students can access allowed material for exams). 
>...
> However, I cannot access any website. In fact, I dont even know, where I should enter the URL?

Place a link to the website you want students to access in your exam and specify that it should be opened in a new browser window (in HTML with the attribute target=_blank or Javascript window.open()). How to place such a link in an exam will depend on the exam system/LMS you're using.

**Reply by AndreGismo** [reply]
ah ok, thank you! So there is no "direct" solution via SEB? I first have to "mess arround" with the LMS, correct?

**Reply by AndreGismo** [reply]
so there is no way to just specify, that SEB should start with directly two browser windows open (where one browser window is the login to the LMS and the other browser is the sciebo site)?

**Reply by danschlet** [reply]
We have such a feature in the backlog of our roadmap (SEB 2.x for Windows already had such a feature, called Additional Resources, but it was outside the scope of the refactoring, so it was dropped for now from 3.x). As there are many competing feature requests and paying stakeholders (see [SEB Alliance](https://safeexambrowser.org/alliance/members.html)) don't prioritize this feature, it's not clear when and if we will implement it.

It's generally extremely easy to add a link to another website to any reasonably designed web application. Therefore I don't see why this should be a big issue to do it on the web application side.
